lib/template/details-json.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+$aPlaceDetails = $aPointDetails;
+
+$aPlaceDetails['geojson'] = json_decode($aPointDetails['asgeojson']);
+unset($aPlaceDetails['asgeojson']);
```

Comment:
Can we make the geometry optional as well for json, i.e. a URL parameter which is off by default? Preferably use the usual polygon_geojson parameter.